genset_whisperpower_ddc.orogen

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+name "genset_whisperpower_ddc"
+
+import_types_from "std"
+import_types_from "base"
+
+using_library "genset_whisperpower_ddc"
+
+import_types_from "genset_whisperpower_ddc/ControlCommand.hpp"
+import_types_from "genset_whisperpower_ddc/GeneratorState.hpp"
+import_types_from "genset_whisperpower_ddc/RunTimeState.hpp"
+
+# If new data types need to be defined, they have to be put in a separate C++
+# header, and this header will be loaded here
```

Comment:
Please remove the boilerplate (autogenerated) comments, and replace them with something meaningful. 

(There are more in this file)